elytron/src/main/java/org/wildfly/extension/elytron/RoleDecoderDefinitions.java

```diff
@@ -53,10 +61,39 @@
         .setRestartAllServices()
         .build();
 
+    static final SimpleAttributeDefinition SOURCE_ADDRESS = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.SOURCE_ADDRESS, ModelType.STRING)
+            .setAllowExpression(true)
+            .setMinSize(1)
+            .setRestartAllServices()
+            .setAlternatives(ElytronDescriptionConstants.PATTERN)
+            .build();
+
+    static final SimpleAttributeDefinition PATTERN = new SimpleAttributeDefinitionBuilder(RegexAttributeDefinitions.PATTERN)
+            .setAlternatives(ElytronDescriptionConstants.SOURCE_ADDRESS)
+            .build();
```

Comment:
@fjuma should this one also restart all services?

Reply:
Yes, added that. Thanks.